skbio/tree/_tree.py

```diff
@@ -2743,7 +2755,7 @@ def index_tree(self):
                                 self.children[0].id,
                                 self.children[-1].id))
 
-        return id_index, child_index
+        return id_index, np.vstack(child_index)
```

Comment:
@wasade, was this an interface change? If so, we should add a note under *Backward-incompatible changes [experimental]* in the change log.